httputils/writers.go

```diff
@@ -0,0 +1,13 @@
+package httputils
+
+import (
+	"fmt"
+	"net/http"
+)
+
+// WriteError sets the response status code and write formats the provided message as the
+// response body
+func WriteError(w http.ResponseWriter, statusCode int, msg string, args ...interface{}) {
```

Comment:
Can we use `http.Error` instead because it's a part of stdlib?

Reply:
yeah, i am not sure why i added this here, i might have had a plan to do something with it that never happened

Reply:
👍 Do you think the code in the proxy could change to `http.Error` too?

Reply:
I see now why the utility was original written, the internal utility wraps the string formatting, which is not supported by `http.Error`

```go
func Error(w ResponseWriter, error string, code int)
```

How do you feel about 

```go
// ErrorF sets the response status code and write formats the provided message as the
// response body
func ErrorF(w http.ResponseWriter, statusCode int, msg string, args ...interface{}) {
	http.Error(w, fmt.Sprintf(msg, args...), statusCode)
}
```

Reply:
I have replaced the utility with the `ErrorF` described above